docker/Dockerfile

```diff
@@ -0,0 +1,25 @@
+ARG PYTHON_VERSION=3.9
+
+FROM docker.io/python:${PYTHON_VERSION}-slim
+
+# Install libmagic. We don't need to install libolm, because we're installing it with a
+# wheel from gitlab.matrix.org later, which comes with libolm already compiled.
+RUN apt-get update -qq && apt-get install -qq libmagic1
+
+# Copy the necessary project files into the image.
+COPY setup.cfg pyproject.toml /
+COPY matrix_content_scanner /matrix_content_scanner/
+
+# Install the project and its dependencies, using gitlab.matrix.org as the index. This is
+# so we can get a python-olm wheel, which will have more recent bindings than the current
+# version of python-olm on PyPI, and comes with libolm already compiled.
+RUN pip install --index-url=https://gitlab.matrix.org/api/v4/projects/27/packages/pypi/simple .
```

Comment:
A bit sad that we're not pinning dependencies somehow.

Reply:
Yep - tracking that as https://github.com/matrix-org/matrix-content-scanner-python/issues/32